### ISSUE TITLE
Moz Github infra repositories

### ISSUE THREAD

**Issue by Akhliskun** [Akhliskun/firefox-infra-changelog#1]
We will need to create a list of all github repositories that are being used for firefox-ci and/or could influence the infra. 

A good place to start would be our team's Google Drive Escalation Doc. 

Please post in the following format:
`<Name of repo> - <Link to github repo>`

**Comment by bccrisan** [comment]
Should we just include all repos from mozilla-releng?

**Comment by Akhliskun** [comment]
This is a good question, at which I have a short and long answer:

### Short answer:
**Yes**, anything that can influence Moz-Infra should be on our "track list".

### Long answer:
Probably, yes. My number one question is if we should track abandoned repositories, lets say repositories which haven't got a single commit in 3-6-12 months. Maybe for keeping a full archive/changelog this would be awesome, but I'm thinking if we go this route, and add to one central json/md file all the changes that happened, in lest say the past 5 years, things could get out of control really easily as the length of the changelog would be something out of this world. 

### TL;DR:
For now **yes**, but it's worth coming back to this once we know with the we have to work.

**Comment by bccrisan** [comment]
Took a quick peek at what mozilla-releng has right now in their repositories, and most of them (let's say 28) have been updated in less than 6 months, the rest of them are pretty old.

Those 28 repositories, I can vouch for them, that would be a good start, for the rest I think we should ask to see if there are still in use and worth adding.

**Comment by Akhliskun** [comment]
Thanks for the investigation, 
I think that we can move forward and start creating the list of repositories. 

Please keep them in 2 lists, using MarkDown formatting options. 

```
## Active repositories (last commit not older than 6 months):
* Repo here
* Repo here

## Other repositories (didn't received a commit in 6 months or more):
* Repo here
* Repo here 
```

**Comment by Akhliskun** [comment]
Hey guys, is this still applicable? 
Are the repositories listed here the ones that we need?

**Comment by bccrisan** [comment]
AFAIK, these are the repositories that I know to affect our infrastructure.
## Active repositories that affect Firefox-CI:
* Ship it - https://github.com/mozilla-releng/ship-it
* services - https://github.com/mozilla/release-services
* beetmoverscript - https://github.com/mozilla-releng/beetmoverscript
* addonscript - https://github.com/mozilla-releng/addonscript
* ship it V2 - https://github.com/mozilla-releng/shipit-v2
* build-cloud-tools - https://github.com/mozilla-releng/build-cloud-tools
* build-puppet - https://github.com/mozilla-releng/build-puppet
* shipitscript - https://github.com/mozilla-releng/shipitscript
* bouncerscript - https://github.com/mozilla-releng/bouncerscript
* treescript - https://github.com/mozilla-releng/treescript
* mozapkpublisher - https://github.com/mozilla-releng/mozapkpublisher
* OpenCloudConfig - https://github.com/mozilla-releng/OpenCloudConfig
* scriptworker - https://github.com/mozilla-releng/scriptworker
* pushsnapscript - https://github.com/mozilla-releng/pushsnapscript
* signingscript - https://github.com/mozilla-releng/signingscript
* cot-gpg-keys - https://github.com/mozilla-releng/cot-gpg-keys
* pushapkscript - https://github.com/mozilla-releng/pushapkscript
* balrogscript - https://github.com/mozilla-releng/balrogscript
* funsize - https://github.com/mozilla-releng/funsize
* signtool - https://github.com/mozilla-releng/signtool
* build-tooltool - https://github.com/mozilla/build-tooltool

## Repositories that are under a maybe (need a sanity check):
* pulse-notify - https://github.com/mozilla-releng/pulse-notify
* mozilla-version - https://github.com/mozilla-releng/mozilla-version
* shipitapi - https://github.com/mozilla-releng/shipitapi
* ami-builder - https://github.com/mozilla-releng/ami-builder
* binary-transparency - https://github.com/mozilla-releng/binary-transparency

This is just a list and it may be incomplete.

**Comment by bccrisan** [comment]
All of the above repositories have been added to the `repositories.json` and it will be merged (probably) in the next PR.

**Comment by bccrisan** [comment]
Commit: https://github.com/Akhliskun/firefox-infra-changelog/commit/8d1b167b9f4135f4118b81db63640f336dd6cb86

**Comment by bccrisan** [comment]
This can be closed.
